yt/yt/ytlib/api/native/client_queries.cpp

```diff
@@ -570,6 +802,19 @@ TListQueriesResult TClient::DoListQueries(const TListQueriesOptions& options)
         }
     };
 
+    auto formatAcosInString = [] (std::vector<TString> accessControlObjects) {
+        TStringBuilder builder;
+        for (std::size_t i = 0; i < accessControlObjects.size(); ++i) {
```

Comment:
1) i -> index
2) We usually use signed types, so size_t -> int and aco.size() -> std::ssize(aco)